README.md

```diff
@@ -1,8 +1,50 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Rae Khan . Jarvis Consulting
 
+My name is Rae Khan, I have recently joined Jarvis as a Data Engineer and would like to introduce myself. I have earned a Bachelors of Aerospace Engineering from Ryerson University. Throughout my academic career, I became proficient at MATLAB coding and within the last several years, I have honed my software skills. I have completed courses on platforms such as Udemy and edX in topics such as VBA programming and SQL. In my previous position, I worked as a Python Instructor. I taught various students the basics of Python, such as syntax, data types, and functions. Additionally, I recently completed a Web Development Bootcamp at Lighthouse Labs gaining additional software skills such as JavaScript, HTML, and CSS. I look forward to expanding my software skillset with Jarvis.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** JavaScript, Python, MATLAB, HTML, CSS, VBA
+
+**Familiar:** Ruby, React, JQuery, Ajax, Express
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_RaeKhan](https://github.com/jarviscanada/jarvis_data_eng_RaeKhan)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_RaeKhan/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_RaeKhan/tree/master/linux_sql)]: The purpose of this project is to monitor a cluster of servers running CentOS 7 for the Jarvis Linux Cluster Administration (LCA) Team. Data is obtained and inserted in an RDBMS database using PostgreSQL, Docker, crontab, and Bash.
```

Comment:
**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_RaeKhan/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_RaeKhan/tree/master/linux_sql)]: The purpose of this project is to monitor a cluster of servers running CentOS 7 for the Jarvis Linux Cluster Administration (LCA) Team. Data is obtained and inserted in an RDBMS database using PostgreSQL, Docker, crontab, and Bash.

Developed xyz for abc purposes using A, B, C. Features, tech keywords... bash script, data -> PSQL, DA using SQL. Testing, packging docker.